### DISCUSSION TITLE
如何将中文版页眉中的`Chapter x:`修改为`第x章`

### DISCUSSION THREAD

**Discussion by He-Jingkai** [sjtug/SJTUThesis#935] [提问] [answered]
主线版本默认设置编译效果：
![image](https://user-images.githubusercontent.com/72253004/234874753-efcd353a-ed04-40cc-b50b-1cdd02f62be4.png)

**Comment by LogCreative** [comment]
检查一下是否使用了 `lang=en` 文档类选项。

**Reply by He-Jingkai** [reply]
未设置，且设置lang=zh后无变化

**Reply by He-Jingkai** [reply]
旧版本无此问题，更新到最新版本后出现

**Comment by He-Jingkai** [comment, marked as answer]
使用debian系统编译发现这个问题，mac os系统编译正常，其他系统未测试

**目测是latexmk版本问题**

**Reply by note286** [reply]
你需要检查一下TeX Live版本，`tex -v`。
